docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -26,13 +27,20 @@ activate FindCommandParser
 FindCommandParser --> GradPadParser
 deactivate FindCommandParser
 
-GradPadParser -> FindCommandParser : parse(" CS2103T")
+GradPadParser -> FindCommandParser : parse("cs foundation")
 activate FindCommandParser
 
 create FindCommand
 FindCommandParser -> FindCommand
 activate FindCommand
 
+create CompoundFindPredicate
+FindCommand -> CompoundFindPredicate
+activate CompoundFindPredicate
+
+CompoundFindPredicate --> FindCommand
+deactivate CompoundFindPredicate
+
```

Comment:
then for line 59, might be better to change `updateFilteredModuleList()` to `updateFilteredModuleList(predicate)` for more clarity